lntest/itest/lnd_routing_test.go

```diff
@@ -1142,10 +1142,9 @@ func testInvoiceRoutingHints(net *lntest.NetworkHarness, t *harnessTest) {
 	}
 	for i, chanPoint := range aliceChans {
 		err := net.Alice.WaitForNetworkChannelOpen(chanPoint)
-		if err != nil {
-			t.Fatalf("timed out waiting for channel open %s: %v",
-				chanNames[i], err)
-		}
+		require.NoErrorf(t.t, err,
```

Comment:
Could you drop this commit? There's another ongoing big refactor #6825 which will change them altogether, thanks!